charts/ingress-nginx/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 3.10.1
+version: 3.11.0
```

Comment:
Please replace [line 19](https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/Chart.yaml#L19) with
`- Support Keda Autoscaling`

and add the change here https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/CHANGELOG.md#unreleased